src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx

```diff
@@ -257,14 +258,18 @@ export default function EnvTriggerView({ filteredAppIds, isVirtualEnv }: AppGrou
             setWorkflows(_workflows)
             setFilteredCIPipelines(filteredCIPipelines)
             setErrorCode(0)
-            setPageViewType(ViewType.FORM)
+            
             getWorkflowStatusData(_workflows)
             processFilteredData(_workflows)
+        }
         } catch (error) {
             showError(error)
             setErrorCode(error['code'])
             setPageViewType(ViewType.ERROR)
         }
+        finally{
+            setPageViewType(ViewType.FORM)
```

Comment:
this will run in case of error as well so there will be no error state. Is it expected?

Reply:
Handled it